timur/test/javascript/helpers.js

```diff
@@ -25,8 +25,6 @@ export const mockStore = configureMockStore([
   thunk
 ]);
 
-export const setConfig = (config) => global.TIMUR_CONFIG = config;
```

Comment:
Added this to the jest.config.js just so that each test "gets this for free" without needing to  explicitly set it.  We have no tests that dynamically need interesting configs; they all basically just assume "some" static values.